pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -37,17 +37,20 @@ type AnacondaInstallerISOTree struct {
 	// Anaconda pipeline.
 	KSPath string
 
-	// The path where the payload (tarball or ostree repo) will be stored.
+	// The path where the payload (tarball, ostree repo, or container) will be stored.
 	PayloadPath string
 
 	isoLabel string
 
 	SquashfsCompression string
 
-	OSPipeline         *OS
-	OSTreeCommitSource *ostree.SourceSpec
+	OSPipeline *OS
 
-	ostreeCommitSpec *ostree.CommitSpec
+	CommitSource *ostree.SourceSpec
```

Comment:
Why the rename?  Not really that important, but I think the ostree prefix is useful here.  A "commit" doesn't really mean much on its own and might be misunderstood out of context.

Reply:
Agreed, I think I was making it consistent with `container*`.